layers/+tools/shadowenv/README.org

```diff
@@ -0,0 +1,31 @@
+#+TITLE: shadowenv layer
+
+#+TAGS: layer|tool
+
+[[file:img/shadowenv.png]]
```

Comment:
I suggest to write copyright notice for all image files from now on.

So add a footnote `[[file:img/shadowenv.png]] [fn:1]`
And `[fn:1] This logo image consists only of simple geometric shapes or text. It does not meet the threshold of originality needed for copyright protection, and is therefore in the public domain.`

Reply:
:+1: done.